smooth_weighted.go

```diff
@@ -66,23 +66,14 @@ func (w *SW) All() map[interface{}]int {
 
 // Next returns next selected server.
 func (w *SW) Next() interface{} {
-	i := w.nextWeighted()
```

Comment:
why removed this line?

Reply:
because can be `return nextSmoothWeighted(w.items).Item`
look blow:    
```
// Next returns next selected server.
func (w *SW) Next() interface{} {
	switch w.n {
	case 0:
		return nil
	case 1:
		return w.items[0].Item
	default:
		return nextSmoothWeighted(w.items).Item
	}
}
```